packages/commonwealth/server/router.ts

```diff
@@ -270,24 +269,40 @@ function setupRouter(
   router.post(
     '/createThread',
     passport.authenticate('jwt', { session: false }),
-    createThread.bind(this, models, tokenBalanceCache, ruleCache, banCache)
-  );
-  router.put(
-    '/editThread',
-    passport.authenticate('jwt', { session: false }),
-    editThread.bind(this, models, banCache)
-  );
-
-  router.post(
-    '/createPoll',
-    passport.authenticate('jwt', { session: false }),
-    createPoll.bind(this, models)
-  );
+    [
+      DatabaseValidationService.validateAuthor.bind(
```

Comment:
Would be nice if we could make these functions cleaner somehow -- maybe your original suggestion was best, of having the service own models.

Reply:
yeah I think maybe we initialize those service owning models in the server. I can play around with this